roles/openshift_sdn/files/sdn-ovs.yaml

```diff
@@ -100,7 +105,7 @@ spec:
           readOnly: true
         - mountPath: /etc/openvswitch
           name: host-config-openvswitch
-        - mountPath: /var/log/openvswitch
+        - mountPath: /var/log/openvswitch-old
```

Comment:
You also need to change the volume type to `DirectoryOrCreate` - see  https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

Reply:
Added DirectoryOrCreate